Zend/zend_ast.c

```diff
@@ -1124,12 +1222,12 @@ static zend_result ZEND_FASTCALL zend_ast_evaluate_inner(
 
 						if (!(fptr->common.fn_flags & ZEND_ACC_STATIC)) {
 							zend_non_static_method_call(fptr);
-							
+
```

Comment:
Whoops. Sorry for introducing these. Can you push the a commit to trim the trailing spaces straight to master to denoise the diff?